integration-cli/docker_cli_start_test.go

```diff
@@ -75,32 +75,26 @@ func (s *DockerSuite) TestStartRecordError(c *check.C) {
 	// when container runs successfully, we should not have state.Error
 	dockerCmd(c, "run", "-d", "-p", "9999:9999", "--name", "test", "busybox", "top")
 	stateErr, err := inspectField("test", "State.Error")
-	c.Assert(err, check.IsNil)
-	if stateErr != "" {
-		c.Fatalf("Expected to not have state error but got state.Error(%q)", stateErr)
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("stateErr: %s", stateErr))
```

Comment:
Just don't want to lose any useful information, I just looked more deeply in the code, seems you are right, it's not necessary, will remove it.